templates/component/src/index.js

```diff
@@ -0,0 +1,8 @@
+import { tag } from '@storefront/core';
+
+@tag('<%= slug %>', require('./<%= slug %>.html'), require('./<%= slug %>.html'))
+export class <%= sanitizedName %> {
+
```

Comment:
Should we add the lifecycle methods here by default?

Reply:
To be honest, I think that would be useful.

Reply:
This is an interesting add, I think that it would help people creating riot components to know about what lifecycle methods are available in the framework. But it also leads to more clean up, if they don't need all of them or any for that matter.

Reply:
implemented, resolving.